### PR TITLE
Add a new `blur-roberts` example

### PR DESCRIPTION
**Pull request by ffevotte**
This implements the tiled application of image filters

### CODE REVIEW

**Comment by ffevotte** [comment]
@maltezfaria Would you like to review this in the PR, or should I go ahead and merge the branch so that the documentation it generates is published? (of course, in the latter case you're still welcome to do any modification you like afterwards)

File: docs/src/examples/blur-roberts/blur-roberts.jl [hunk 1]
```diff
@@ -0,0 +1,300 @@
...
+#=
+
+## Filters implementation
+
+The `blur!` function averages the value of each pixel with the values of all
+pixels less than `width` pixels away in manhattan distance. In order to simplify
+the implementation, the filter is applied only to pixels that are sufficiently
+far from the boundary to have all their neighbors correctly defined.
+
+Results are written in-place a pre-allocated `dest` array. Unless otherwise
```
Comment (maltezfaria) [inline docs/src/examples/blur-roberts/blur-roberts.jl]: Typo?

**Review by maltezfaria — APPROVED** [review]

**Comment by maltezfaria** [comment]
> @maltezfaria Would you like to review this in the PR, or should I go ahead and merge the branch so that the documentation it generates is published? (of course, in the latter case you're still welcome to do any modification you like afterwards)

LGTM.

**Comment by ffevotte** [comment]
Thanks!

**Pull request merged**